src/server.h

```diff
@@ -366,16 +372,13 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 		friend class TestServerShutdownState;
 		public:
 			bool is_requested = false;
-			bool should_reconnect = false;
-			std::string message;
+			ShutdownInformation info{.should_reconnect = false};
```

Comment:
spacing

Reply:
What's correct spacing?

Reply:
Good question. The spacing rules within brackets are only mentioned in the Lua code style. I'd imagine either
`ShutdownInformation info {.should_reconnect = false};`
or
```C++
ShutdownInformation info {
	.should_reconnect = false
};
```

Reply:
fixed